### PR TITLE
fix #109136, fix #33201: add some articulations

### LINKED ISSUE #33201: Де в новій версії MuseScore є розділ "інструменти", коли вже створену партитуру і хочеш прибрати певні інструменти. І де є розділ партії щоб зробити партії?
Де в новій версії MuseScore є розділ "інструменти", коли вже створену партитуру і хочеш прибрати певні інструменти. І де є розділ партії щоб зробити партії?

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
stringHarmonics and a few fermatas as well as getting these and the ones from dafb5f3 and ab354df translated. Also getting some of the previously added articulations to play back properly and import from OVE as well as export to XML (not complete though, and import is missing)

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
This would make #2826 obsolete, so I'll close that now

File: libmscore/sym.cpp [hunk 1]
```diff
@@ -2895,62 +2895,63 @@ QVector<QString> Sym::symUserNames = {
...
-      "Combining ricochet for stem (3 tones)",
-      "Combining ricochet for stem (4 tones)",
-      "Combining ricochet for stem (5 tones)",
-      "Combining ricochet for stem (6 tones)",
+      QT_TRANSLATE_NOOP("symUserNames", "Combining accordion coupler dot"),
+      QT_TRANSLATE_NOOP("symUserNames", "Combining left hand, 2 ranks, empty"),
+      QT_TRANSLATE_NOOP("symUserNames", "Combining left hand, 3 ranks, empty (square)"),
+      QT_TRANSLATE_NOOP("symUserNames", "Combining right hand, 3 ranks, empty"),
+      QT_TRANSLATE_NOOP("symUserNames", "Combining right hand, 4 ranks, empty"),
+      QT_TRANSLATE_NOOP("symUserNames", "Diatonic accordion clef",
```
Comment (lasconic) [inline libmscore/sym.cpp]: missing ) here.

Comment (Jojo-Schmitz) [inline libmscore/sym.cpp]: fixed, sorry about that

**Comment by lasconic** [comment]
It needs rebase but it looks good.

A couple of remarks/questions
* Why the TODO for harmonics in MusicXML ? https://github.com/musescore/MuseScore/pull/2854/files#diff-aa65d42ca303b9e5f36988b9779a30acR2167
* I believe a MusicXML 3.1 will support SMuFL names for articulation so it would solve a lot of problems in the MusicXML import/export

**Comment by Jojo-Schmitz** [comment]
One TODO is bec

**Comment by Jojo-Schmitz** [comment]
One TODO is because I'm not 100% sure it is correct, the other because I'm not 100% sure it is wrong and on both cases would like @lvinken 's feedback. Will look into rendering it coming Sunday

**Comment by Jojo-Schmitz** [comment]
rebased and made one TODO active (the one where I'm pretty sure to have done the right thing, esp. now after having fixed a typo in it)

**Pull request merged**